test/pde_systems/burgers_eq.jl

```diff
@@ -91,93 +144,92 @@ end
 end
 
 # Exact solutions from: https://www.sciencedirect.com/science/article/pii/S0898122110003883
-@testset "Test 01: Burger's Equation 2D" begin
-    @parameters x y t
-    @variables u(..) v(..)
-    Dt = Differential(t)
-    Dx = Differential(x)
-    Dy = Differential(y)
-    Dxx = Differential(x)^2
-    Dyy = Differential(y)^2
+# @testset "Test 01: Burger's Equation 2D" begin
```

Comment:
?

Reply:
This has never passed, I'm not sure its even correct.